arches_controlled_lists/src/arches_controlled_lists/components/editor/ListCharacteristic.vue

```diff
@@ -76,70 +76,94 @@ const cancel = () => {
 
 <template>
     <div class="characteristic">
-        <h4>{{ props.label }}</h4>
-        <!-- TODO https://github.com/archesproject/arches/issues/10847 -->
-        <span
-            v-if="!props.editable"
-            style="font-size: small"
-        >
-            {{ $gettext("False") }}
-        </span>
+        <div>
+            <div class="value-editor-title">
+                <h4>{{ props.label }}</h4>
+            </div>
+            
+            <!-- TODO https://github.com/archesproject/arches/issues/10847 -->
+            <span class="value-label"
```

Comment:
Vue linters are picky about component shape and attribute order. Can this be changed to:

```
            <span 
                v-if="!props.editable"
                class="value-label"
                style="font-size: small"
            >
```